src/__tests__/macro.test.js

```diff
@@ -84,6 +86,25 @@ pluginTester({
         process.env.NODE_PATH = 'src/';
       },
     },
+    '[loader] with absolute path and jsconfig include property': {
+      error: false,
+      code: `
+        import { loader } from '../macro';
+        const query = loader('__tests__/fixtures/simpleFragment.graphql');
+      `,
+      setup: () => {
```

Comment:
I couldn't find any alternative way to test this feature, here I symlink the `jsconfig.json` file (stored in the fixtures folder) to the root of the project, run the test, and then delete the symlink

Reply:
LGTM